airflow/bin/cli.py

```diff
@@ -993,6 +1047,38 @@ class CLIFactory(object):
             ("-x", "--delete"),
             metavar="KEY",
             help="Delete a variable"),
+        # connections
+        'create': Arg(
+            ("-c", "--create"), "Create a new connection", "store_true"),
+        'delete': Arg(
+            ("-d", "--delete"), "Delete one or more connections", "store_true"),
+        'long': Arg(
+            ("-l", "--long"), "Display connection info in details", "store_true"),
+        'no_prompt': Arg(
+            ("-np", "--no_prompt"), "Do not prompt to confirm connection deletion",
+            "store_true"),
+        'conn_id': Arg(
+            ("-ci", "--conn_id"), "Set/filter by the connection identifier"),
+        'conn_type': Arg(
+            ("-ct", "--conn_type"),
+            choices=['bigquery', 'cloudant', 'google_cloud_platform', 'hive_cli',
+                     'hiveserver2', 'jdbc', 'mssql', 'mysql', 'oracle', 'postgres',
+                     'presto', 'sqlite', 'vertica'],
```

Comment:
in `views.py` we use similar list to show connection drop down, it would be better if we maintain a single list (may be in `Connection` model) and use at both places. So in future ppl don't have to rmbr multiple places for mentioning their connections.